torch/_inductor/fx_passes/pre_grad.py

```diff
@@ -621,12 +622,15 @@ def one_user(node):
         return users[0] if len(users) == 1 else None
 
     def is_view(node):
-        view = {"view"}
+        view = OrderedSet(["view"])
         return node.op == "call_method" and node.target in view
 
     def is_pointwise_unary(node):
-        pointwise = {torch.relu, torch.tanh, "relu", "tanh"}
```

Comment:
see, this example too..

Reply:
I changed these to be just tuples. The benchmark above showed that testing membership in small tuples is significantly faster than in `OrderedSet`, and not much slower than `set`.

The problem is that it's fairly hard to automatically detect if a set is really never used for iteration, anyway. A lot of these small sets are public members or even global variables, where it's essentially impossible to detect this.